test/coverlet.core.tests/Reporters/Reporters.cs

```diff
@@ -30,13 +30,6 @@ public class Reporters
         public void Msbuild_ReportWriter(string coverletMultiTargetFrameworksCurrentTFM, string coverletOutput, string reportFormat, string expectedFileName)
```

Comment:
Maybe add a new test instead of changing this one

Reply:
I'm not sure what new test should be written. The purpose of the `Msbuild_ReportWriter` test is to ensure a proper report path. The change I introduced, i.e. returning the path of the report in the `WriteReport` method instead of discarding it just allowed to simplify that test by not intercepting the `WriteAllText` call and use the returned value.

Reply:
@tonerdo I think it's ok the tests is for `ReportWriter` and return path check is in the context of test.